semeio/jobs/rft/trajectory.py

```diff
@@ -0,0 +1,69 @@
+import sys
+import os
+
+from semeio.jobs.rft.utility import strip_comments
+
+
+class TrajectoryPoint:
+    def __init__(self, utm_x, utm_y, measured_depth, true_vertical_depth, zone=None):
+        self.utm_x = utm_x
+        self.utm_y = utm_y
+        self.measured_depth = measured_depth
+        self.true_vertical_depth = true_vertical_depth
+        self.zone = zone
+        self.grid_ijk = None
+        self.pressure = None
+        self.valid_zone = False
+
+    def set_ijk(self, point):
+        self.grid_ijk = point
+
+    def validate_zone(self, zonemap):
+        if self.zone is None:
+            self.valid_zone = True
+        elif self.grid_ijk:
+            self.valid_zone = zonemap is None or zonemap.has_relationship(
+                self.zone, self.grid_ijk[2]
+            )
+
+    def is_active(self):
+        return (
+            self.grid_ijk is not None and self.pressure is not None and self.valid_zone
+        )
+
+    def inactive_info(self, zonemap):
```

Comment:
what happens if `grid_ijk is not None` and `pressure is not None` and `valid_zone`. i guess nothing - and - is that ok?